File: python-selenium/car_data_utility.py
import random
import string

def generate_random_trim_level():
    random_trim_level = ''.join([random.choice(string.ascii_letters + string.digits) for n in range(0, 5)])

    return random_trim_level

def get_car_string(car):
    return car['year'] + " " + car['make'] + " " + car['model'] + " With trim levels: " + ",".join(car['trim_levels'])

File: java-car-data/car-data-web-ui-tests/src/test/java/com/glicerial/samples/cardata/web/uitests/CarDataUtility.java
package com.glicerial.samples.cardata.web.uitests;

import static org.apache.commons.text.CharacterPredicates.DIGITS;
import static org.apache.commons.text.CharacterPredicates.LETTERS;

import java.util.Map;

import org.apache.commons.text.RandomStringGenerator;

public class CarDataUtility {

    public String generateRandomTrimLevel() {
        int length = 5;
        RandomStringGenerator generator = new RandomStringGenerator.Builder()
            .withinRange('0', 'z')
            .filteredBy(LETTERS, DIGITS)
            .build();

        return  generator.generate(length);
    }

    public String getCarString(Map<String, String> carMap) {
        return carMap.get("year") + " " +
            carMap.get("make") + " " +
            carMap.get("model") +
            " With trim levels: " + carMap.get("trimLevels").replaceAll("\n", ",");
    }
}
